commons/src/main/java/org/frankframework/util/StreamUtil.java

```diff
@@ -45,16 +45,15 @@
 
 /**
  * Functions to read and write from one stream to another.
- *
+ * Be careful: Util classes should NOT depend on the Servlet-API
  * @author  Gerrit van Brakel
  */
-//Be careful: UTIL classes should NOT depend on the Servlet-API
 public class StreamUtil {
 
 	public static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
 	public static final String DEFAULT_INPUT_STREAM_ENCODING=DEFAULT_CHARSET.displayName();
 	public static final String AUTO_DETECT_CHARSET = "auto";
-	public static final int BUFFERSIZE=20000;
+	public static final int BUFFER_SIZE = 20_000;
```

Comment:
Ik heb een vermoeden dat I/O buffers nog steeds het beste werken als het veelvouden van 1024 (of misschien zelfs 4096) zijn. Zullen we dat maar gelijk aanpassen naar een mooi veelvoud van 4096?

Reply:
Ik heb even Google Gemini geraadpleegd en mijn conclusie is dat als er iets met disk/memory blocks te maken heeft het handig kan zijn. Echter zijn die groottes anders (groter) dan vroeger... en soms kun je zelfs met de CPU cache size rekening houden.. Anyways, ik maak er wel 64KB van. Mooie factor van 4 en zelfs 8KB.